<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pB_search">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lE_port"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="hL_bR">
            <item>
             <widget class="QLabel" name="l_bR">
              <property name="text">
               <string>BaudRate (Скорость):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cB_bR">
              <property name="currentIndex">
               <number>3</number>
              </property>
              <item>
               <property name="text">
                <string>1200</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2400</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4800</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>9600</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>19200</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>38400</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>57600</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>115200</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hL_dB">
            <item>
             <widget class="QLabel" name="l_dB">
              <property name="text">
               <string>DataBits (Количество битов информации):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cB_dB">
              <property name="currentIndex">
               <number>3</number>
              </property>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>7</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>8</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hL_P">
            <item>
             <widget class="QLabel" name="l_P">
              <property name="text">
               <string>Parity (Проверка на четность):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cB_P">
              <item>
               <property name="text">
                <string>NoParity (Нет бита четности)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>EvenParity (1 бит четности, общее количество битов всегда четное)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>OddParity (1 бит четности, общее количество битов всегда нечетное)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SpaceParity (Бит четности в состоянии сигнала пробела, нет информации об ошибках)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MarkParity</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hL_sB">
            <item>
             <widget class="QLabel" name="l_sB">
              <property name="text">
               <string>StopBits (Количество стоп-битов):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cB_sB">
              <item>
               <property name="text">
                <string>OneStop (1 стоповый бит)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>OneAndHalfStop (1,5 стоповых бита)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>TwoStop (2 стоповых бита)</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hL_fC">
            <item>
             <widget class="QLabel" name="l_fC">
              <property name="text">
               <string>FlowControl (Контроль передачи):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cB_fC">
              <item>
               <property name="text">
                <string>NoFlowControl (Нет контроля потока)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>HardwareControl (Аппаратное управление потоком RTS/CTS)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SoftwareControl (Программное управление потоком XON/XOFF)</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pB_SaveSett">
        <property name="text">
         <string>Сохранить настройки</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="pB_control">
        <item>
         <widget class="QPushButton" name="pB_Con">
          <property name="text">
           <string>Подключиться</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pB_Discon">
          <property name="text">
           <string>Отключиться</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="l_sendText">
            <property name="text">
             <string>Отправить текст:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lE_sendText"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pB_float">
          <property name="text">
           <string>Вывести число в формате float</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pB_sendData">
          <property name="text">
           <string>Отправить в МК</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="tE_receive"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>965</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
